### ISSUE TITLE
T1202 Test#2: "ERROR: The system cannot find the file specified."

### ISSUE THREAD

**Issue by cccs-richardh** [redcanaryco/atomic-red-team#1701]
## What did you do?
I ran the following command using the invoke-atomicredteam package:
`Invoke-AtomicTest T1202 -TestNumbers 2`

## What did you expect to happen?
I expected that the calc.exe would pop.

## What happened instead?
calc.exe does indeed run
BUT
I also get an "ERROR: The system cannot find the file specified

## Your Environment
I'm running Win 10: 10.0.18363 Build 18363 on a 64 bit VM.
I'm using Powershell ^ Administrator
This issue is specifically with T1202 Test #2. It causes an error to be raised about a missing file.

## My notes
The problem with this test is pretty simple to fix. The yaml file contains this:
```
 name: Indirect Command Execution - forfiles.exe
  auto_generated_guid: 8b34a448-40d9-4fc3-a8c8-4bb286faf7dc
  description: |
    forfiles.exe may invoke the execution of programs and commands from a Command-Line Interface.
    [Reference](https://github.com/api0cradle/LOLBAS/blob/master/OSBinaries/Forfiles.md)
    "This is basically saying for each occurrence of notepad.exe in c:\windows\system32 run calc.exe"
    Upon execution calc.exe will be opened
  supported_platforms:
  - windows
  input_arguments:
    process:
      description: Process to execute
      type: String
      default: calc.exe
  executor:
    command: |
      forfiles /p c:\windows\system32 /m notepad.exe /c #{process}
>>>>>      forfiles /p c:\windows\system32 /m notepad.exe /c "c:\folder\normal.dll:evil.exe"  <<<<<
    name: command_prompt
```

The indicated line above (with evil.exe) is the problem.

This test is trying to do both of these malicious examples BUT the c:\folder\normal.dll:evil.exe file
simply doesn't exist.

The author found these examples here: https://github.com/api0cradle/LOLBAS/blob/master/OSBinaries/Forfiles.md 
where there isn't much documentation. 
The NEW location for this info is in the LOLBAS repo, here: 
https://github.com/LOLBAS-Project/LOLBAS/blob/23dd0236aeb756dfccd389827eafecf1639cb40c/yml/OSBinaries/Forfiles.yml
In this updated repo, there is a bit more info about each of the attacks. 
This updated repo also DOES NOT contain an evil.exe or normal.dll (that I could see anyways).

**To fix**, we just need to remove the offending line OR provide a normal.dll/evil.exe in a T1202/src folder.

To be clear, this test works, it's just that the second command is cruft, it does nothing and just puts errors into the log, if you check it. 8-D

**Comment by cccs-richardh** [comment]
Score! I have issue 1701! I'm going boldly where no malicious tester has gone before!

**Comment by clr2of8** [comment]
great info, thanks for posting and we'll get it fixed!

**Comment by clr2of8** [comment]
fixed in PR #1704
